cudl-production/terraform.tfvars

```diff
@@ -19,19 +18,37 @@ lambda-db-secret-key         = "production/cudl/cudl_viewer_db"
 source-bucket-sns-notifications = [
```

Comment:
Would it make sense to rename this variable to `transform-lambda-bucket-sns-notifications` in the `cudl-data-processing` module as the existing name is confusing given the list can now also contain the dest bucket?

Reply:
Renamed.